File: app/src/main/java/com/example/homeworkproject/client/RetrofitClient.java
package com.example.homeworkproject.client;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitClient {

    private static final Retrofit.Builder retrofitBuilder =
            new Retrofit.Builder()
                .baseUrl(Api.BASE_URL)
                .addConverterFactory(GsonConverterFactory.create());

    private static final Retrofit retrofit = retrofitBuilder.build();

    private static final Api myApi = retrofit.create(Api.class);

    public static Api getMyApi(){
        return myApi;
    }
}

File: app/src/main/java/com/example/homeworkproject/client/RetrofitClient.kt
package com.example.homeworkproject.client

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object RetrofitClient {

    private var BASE_URL:String = "https://connect.mindbodyonline.com/rest/worldregions/"
    private val retrofitBuilder: Retrofit.Builder = Retrofit.Builder()
        .baseUrl(BASE_URL)
        .addConverterFactory(GsonConverterFactory.create())
    private val retrofit: Retrofit = retrofitBuilder.build()

    val myApi: Api = retrofit.create(Api::class.java)
}
